Document frida-server filename detection (#257)
frida/frida-website: _i18n/en/_docs/android.md

$ adb shell "/data/local/tmp/frida-server &"
{% endhighlight %}

> Some apps might be able to detect frida-server location. Change the frida-server binary to a random name, or moving the frida-server binary to another location such as /dev should work.

Reviewer: ```suggestion
Some apps might be able to detect the frida-server location. Renaming the
frida-server binary to a random name, or moving it to another location
such as /dev may do the trick.
```
